config/default/common/config/metadata/stories/black_marble_night_lights/step001.md

```diff
@@ -0,0 +1,3 @@
+The Day/Night Band allows scientists to monitor daily various nighttime phenomena on the Earth's surface and atmosphere in low light conditions. This Black Marble Nighttime At Sensor Radiance layer is created from the Day/Night Band-based at-sensor top-of-atmosphere night time radiance product (VNP46A1).
```

Comment:
--> "The Day/Night Band allows scientists to monitor various nighttime phenomena on the Earth's surface and atmosphere in low light conditions on a daily basis."

Reply:
i was struggling with that one for some reason! thanks for making it better :D